zulipterminal/config/keys.py

```diff
@@ -35,27 +43,27 @@
     },
     'REPLY_MESSAGE': {
         'keys': {'r', 'enter'},
-        'help_text': 'Reply to a message',
-    },
-    'REPLY_AUTHOR': {
-        'keys': {'R'},
-        'help_text': 'Reply to an author',
+        'help_text': 'Reply to current message',
     },
     'MENTION_REPLY': {
         'keys': {'@'},
-        'help_text': 'Reply mentioning the sender of the message'
+        'help_text': 'Reply mentioning the sender of the current message'
     },
     'QUOTE_REPLY': {
         'keys': {'>'},
-        'help_text': 'Reply quoting message text',
+        'help_text': 'Reply quoting current message text',
+    },
+    'REPLY_AUTHOR': {
+        'keys': {'R'},
+        'help_text': 'Reply privately to the sender of the current message',
     },
     'STREAM_MESSAGE': {
         'keys': {'c'},
-        'help_text': 'New stream message',
+        'help_text': 'New message to a stream',
     },
     'PRIVATE_MESSAGE': {
         'keys': {'x'},
-        'help_text': 'New private message',
+        'help_text': 'New message to a person',
```

Comment:
"New message to a person or group of people"?

Reply:
@rishig Good catch - I agree about this and have changed it in the PR. I wanted to check this in the code before changing, but it has exposed the way in which we handle PMs, which I think might need more work too (emails vs names).